package.json

```diff
@@ -27,12 +27,14 @@
     "esbuild": "0.8.55",
     "jasmine": "3.6.3",
     "npm-run-all": "4.1.5",
-    "onchange": "7.1.0",
     "playwright": "1.9.1",
     "sirv": "1.0.11",
     "ts-node": "9.1.0",
     "typescript": "4.1.2"
   },
+  "optionalDependencies": {
+    "onchange": "7.1.0"
```

Comment:
`onchange` fails the `npm install` on Windows because of `fsevents`. Marking it as optional solves the problem.